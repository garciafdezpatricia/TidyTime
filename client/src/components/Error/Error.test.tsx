import { render } from "@testing-library/react";
import '@testing-library/jest-dom';
import ErrorPage from '@/src/components/Error/Error';

describe('Error', () => {
    it('renders without crashing', () => {
      const { container } = render(<ErrorPage error={new Error()} resetErrorBoundary={undefined}/>);
      expect(container).toBeInTheDocument();
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<ErrorPage error={new Error()} resetErrorBoundary={undefined}/>);
        expect(asFragment()).toMatchSnapshot();
      });
  });