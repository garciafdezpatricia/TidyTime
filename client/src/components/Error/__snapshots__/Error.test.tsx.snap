// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="alert"
  >
    <div
      class="error-img"
    />
    <h1>
      errorPage.title
    </h1>
    <p
      class="error-p"
    >
      errorPage.desc
      <a
        href="https://github.com/garciafdezpatricia/TidyTime/issues"
      >
        errorPage.link
      </a>
    </p>
    <button
      class="refresh-page"
    >
      errorPage.refreshButton
    </button>
  </div>
</DocumentFragment>
`;
